README.md

```diff
@@ -35,7 +35,15 @@ This site is built using [Quarto](https://quarto.org/docs/get-started/). To prev
 
 This project is licensed under the Creative Commons Attribution 4.0 International license.
 
-Preferred citation: `Barciauskas, A et al. 2023. Cloud Optimized Geospatial Formats Guide. CC-By-4.0`.
+## Citation
+
+Please cite the Cloud Optimized Geospatial Formats Guide through the project’s Zenodo archive using DOI: <TBA>. This DOI represents all versions, and will always resolve to the latest one.
+
+The citation will look something like:
+
+> Aimee Barciauskas, TBA (2023). cloudnativegeo/cloud-optimized-geospatial-formats-guide: Cloud Optimized Geospatial Formats Guide v2025.05.21 (v2025.05.21). Zenodo. <TBA>
```

Comment:
```suggestion
>Aimee Barciauskas, Kyle Barron, Zac Deziel, Alex I. Mandel, Jed Sundwall, Julia Signell, Max Jones, Jonas, Leo Thomas, Jamison French, Tyler Erickson, Rajat Shinde, Kiri, Daniel Sparing, Blayne Chard, Brian Mc Donald, James Bourbeau, Luis López, autarc, & rsignell. (2025). Cloud Optimized Geospatial Formats Guide:  https://doi.org/10.5281/zenodo.15498529
```
This is a slightly edited APA style citation.

Looks like Harvard and MLA are actually close to the current, it's fine to redirect people to the DOI zenodo page for other styles.

Reply:
To be clear I'm copying and pasting from the Citation generator on Zenodo with minor edits.